server/main.cpp

```diff
@@ -18,9 +20,25 @@ using asio::ip::tcp;
 tec::state_id_t current_state_id = 0;
 
 namespace tec {
-	std::map<std::string, std::function<void(std::string)>> file_factories;
-	std::map<tid, std::function<void(const proto::Entity&, const proto::Component&)>> in_functors;
 	eid active_entity;
+	std::string LoadJSON(const FilePath& fname) {
+		std::fstream input(fname.GetNativePath(), std::ios::in | std::ios::binary);
+		std::string in;
+		input.seekg(0, std::ios::end);
+		in.reserve(input.tellg());
+		input.seekg(0, std::ios::beg);
+		std::copy((std::istreambuf_iterator<char>(input)), std::istreambuf_iterator<char>(), std::back_inserter(in));
+		input.close();
+		return std::move(in);
+	}
```

Comment:
define this in common?

Reply:
It was a hacked to get assets into a json format